website/source/guides/operations/vault-integration/index.html.md

```diff
@@ -145,207 +209,293 @@ An example token role definition is given below:
+Task Group  Queued  Starting  Running  Failed  Complete  Lost
+db          0       0         1        0       0         0
+
+Allocations
+ID        Node ID   Task Group  Version  Desired  Status   Created    Modified
+701e2699  5de1330c  db          0        run      running  1m56s ago  1m33s ago
 ```
 
-The `-orphan` flag is included when generating the Nomad server token above to
-prevent revocation of the token when its parent expires. Vault typically
-creates tokens with a parent-child relationship. When an ancestor token is
-revoked, all of its descendant tokens and their associated leases are revoked
-as well.
-
-When generating Nomad's Vault token, we need to ensure that revocation of the
-parent token does not revoke Nomad's token. To prevent this behavior we
-specify the `-orphan` flag when we create the Nomad's Vault token. All
-other tokens generated by Nomad for jobs will be generated using the policy
-default of `orphan = false`.
+Now we can move on to configuring the connection between Vault and our database.
 
-More information about creating orphan tokens can be found in
-[Vault's Token Hierarchies and Orphan Tokens documentation][tokenhierarchy].
+### Step 9: Enable the Database Secrets Engine
 
-The token can then be set in the server configuration's
-[`vault` stanza][config], as a command-line flag, or via an environment
-variable.
+We are using the database secrets engine for Vault in this exercise so that we
+can generate dynamic credentials for our PostgreSQL database. Run the following command to enable it:
 
+```shell
+$ vault secrets enable database
 ```
-$ VAULT_TOKEN=f02f01c2-c0d1-7cb7-6b88-8a14fada58c0 nomad agent -config /path/to/config
+If the previous command was successful, you will see the following output:
+
+```shell
+Success! Enabled the database secrets engine at: database/
 ```
 
-An example of what may be contained in the configuration is shown below. For
-complete documentation please see the [Nomad agent Vault integration][config]
-configuration.
+### Step 10: Configure the Database Secrets Engine
 
-```hcl
-vault {
-  enabled          = true
-  ca_path          = "/etc/certs/ca"
-  cert_file        = "/var/certs/vault.crt"
-  key_file         = "/var/certs/vault.key"
-  address          = "https://vault.service.consul:8200"
-  create_from_role = "nomad-cluster"
+Create a file named `connection.json` and placed the following information into
+it:
+
+```json
+{
+  "plugin_name": "postgresql-database-plugin",
+  "allowed_roles": "accessdb",
+  "connection_url": "postgresql://{{username}}:{{password}}@database.service.consul:5432/postgres?sslmode=disable",
+  "username": "postgres",
+  "password": "postgres123"
 }
 ```
+The information above allows Vault to connect to our database and create users
+with certain privileges. We will specify the `accessdb` role soon. Please note that it is very common to give Vault the root credentials and let Vault manage the auditing and lifecycle of generated credentials.
```

Comment:
It's my fault and I need to update the Vault doc, but please remove **root credentials** from the last sentence.  So, rephrase it to say something like: 
`Note that Vault needs enough privilege to generate database credentials dynamically and manage their lifecycle.`